airflow/dag_processing/manager.py

```diff
@@ -102,10 +102,16 @@ class DagFileStat:
 class DagFileInfo:
     """Information about a DAG file."""
 
-    path: str  # absolute path of the file
+    rel_path: Path
     bundle_name: str
     bundle_path: Path | None = field(compare=False, default=None)
```

Comment:
Is this able to be None for testing reasons? I feel like it should at least not have a default of None. (I feel like I asked this elsewhere, but can't find it...)

Reply:
if you look [here](https://github.com/apache/airflow/pull/46290/files#diff-827b7a469438ffd1b172ae295134a84c8a914c9ea4c9ea2a7d3de1f1d5aa6bb6R281), bundle path is not known, but it is also not _needed_ -- we only use it to look up one in the dict, which we can do without bundle path since it's not needed for comparison